src/variants.ini

```diff
@@ -733,26 +801,171 @@ nMoveRule = 0
 nFoldValue = loss
 stalemateValue = loss
 
-# 5x5 breakthrough
-[breakthrough5:breakthrough]
-maxFile = 5
-maxRank = 5
-startFen = ppppp/ppppp/5/PPPPP/PPPPP w 0 1
-whiteFlag = *5
-blackFlag = *1
-
-# 6x6 breakthrough
-[breakthrough6:breakthrough]
-maxFile = 6
-maxRank = 6
-startFen = pppppp/pppppp/6/6/PPPPPP/PPPPPP w 0 1
-whiteFlag = *6
-blackFlag = *1
+[gorogoroplus:gorogoro]
+startFen = sgkgs/5/1ppp1/1PPP1/5/SGKGS[LNln] w 0 1
+lance = l
+shogiKnight = n
+promotedPieceType = l:g n:g
 
-# 7x7 breakthrough
-[breakthrough7:breakthrough]
+[dragonfly:chess]
 maxFile = 7
 maxRank = 7
-startFen = ppppppp/ppppppp/7/7/7/PPPPPPP/PPPPPPP w 0 1
+pocketSize = 7
+castlingKingsideFile = f
+castlingQueensideFile = b
+pieceDrops = true
+capturesToHand = true
+promotionRank = 7
+promotionPieceTypes = nbr
+doubleStep = false
+dropNoDoubled = p
+dropNoDoubledCount = 0
+startFen = rbbknnr/ppppppp/7/7/7/PPPPPPP/RBBKNNR[] w - - 0 1
+
+[kamikazerooks:chess]
+extinctionValue = win
+extinctionPieceTypes = r
+
+[mounted]
+maxRank = 7
+maxFile = 7
+pieceDrops = true
+capturesToHand = true
+stalemateValue = loss
+nFoldValue = loss
+extinctionValue = loss
+extinctionPseudoRoyal = true
+extinctionPieceTypes = k
+flagPiece = k
 whiteFlag = *7
-blackFlag = *1
\ No newline at end of file
+blackFlag = *1
+centaur = k
+bishop = b
+rook = r
+cannon = c
+customPiece1 = a:mBcpB 
+customPiece2 = s:FsfW
+startFen = 1cbkra1/2s1s2/7/7/7/2S1S2/1CBKRA1 w - - 0 1
+
+[racingchess:chess]
+flagPiece = k
+whiteFlag = *8
+blackFlag = *1
+flagMove = true
+checking = false
+
+[shinobimirror:crazyhouse]
+commoner = c
+bers = d
+archbishop = j
+fers = m
+shogiKnight = h
+lance = l
+promotionRank = 7
+promotionPieceTypes = -
+promotedPieceType = p:c m:b h:n l:r
+mandatoryPiecePromotion = true
+stalemateValue = loss
+perpetualCheckIllegal = true
+startFen = lh1ck1hl/pppppppp/8/8/8/8/PPPPPPPP/LH1CK1HL[LHMMDJlhmmdj] w - 0 1
+capturesToHand = false
+whiteDropRegion = *1 *2 *3 *4
+blackDropRegion = *5 *6 *7 *8
+immobilityIllegal = true
+flagPiece = k
+whiteFlag = *8
+blackFlag = *1
+castling = false
+
+[twokings2:chess]
+# Slightly different implementation of Two Kings chess, playable in Pychess discord server.
+commoner = k
+extinctionValue = loss
+extinctionPseudoRoyal = true
+extinctionPieceTypes = k
+extinctionPieceCount = 1
+startFen = rnbkkbnr/pppppppp/8/8/8/8/PPPPPPPP/RNBKKBNR w - - 0 1
+
+[atomicrk:racingkings]
+startFen = 4BNRQ/4BNRK/8/8/8/8/krnb4/qrnb4 w - - 0 1
+flagPiece = k
+whiteFlag = *1
+blackFlag = *8
+flagMove = true
+castling = false
+checking = false
+blastOnCapture = true
+
+[epicatomic:atomic]
+variantTemplate = atomic
+king = k
+queen = q
+bishop = b
+knight = n
+rook = r
+pawn = p
+promotionPieceTypes = b
+blastOnCapture = true
+castling = false
+startFen = rnnbqkbnnr/pppppppppp/10/10/10/10/PPPPPPPPPP/RNNBQKBNNR[] w KQkq - 0 1
+maxRank = 8
+maxFile = 10
+chess960 = false
+promotionRank = 6
+doubleStep = true
+castling = false
+
+[atomicshogi]
```

Comment:
This looks wrong, at least speaking from the name of the variant. Perhaps the inheritance `:shogi` is just missing. However, even with that it would not work yet, since the usual replacement of king by commoner would still be required to properly turn it into a working atomic variant. I think you at least need to add
```
king = -
commoner = k
extinctionValue = loss
extinctionPieceTypes = k
blastOnCapture = true
extinctionPseudoRoyal = true
```